Scripts/Runtime/Entities/Transform/TransformUtil.cs

```diff
@@ -35,19 +35,9 @@ public static void AddMissingStandardComponents(Entity entity, EntityManager ent
         {
             Debug.Assert(entityManager.Exists(entity));
 
-            if (!entityManager.HasComponent<Translation>(entity))
+            if (!entityManager.HasComponent<LocalTransform>(entity))
```

Comment:
I'm not sure this util method is even required anymore now that it's just one type.
If you just want to make a TODO issue for this that's fine.

Reply:
Added TODO for #302